packages/twitch/src/API/Helix/EventSub/HelixEventSubApi.ts

```diff
@@ -276,4 +276,25 @@ export class HelixEventSubApi extends BaseApi {
 				transport
 			);
 	}
+
+	async subscribeToChannelRewardRemoveEvents(
+		broadcaster: UserIdResolvable,
+		transport: HelixEventSubTransportOptions,
+		rewardId?: string
```

Comment:
Not sure if this is the best way to deal with the optional condition (or even if it's being called correctly in `src/Subscriptions/EventSubChannelRewardRemoveSubscription.ts`)

Reply:
Please create different methods for passing and not passing a `rewardId` here. The transport options should always be last.

You can use this as an example for the subscription (the WebHooks library was very similarly modeled):
https://github.com/d-fischer/twitch/blob/4c8b1be1fafde338e16731f2e5a987ce8af5cd50/packages/twitch-webhooks/src/Subscriptions/BanEventSubscription.ts#L31-L43